schema.js

```diff
@@ -0,0 +1,23 @@
+const summarySchema = new mongoose.Schema({
+    restaurant_id: {type: Number, unique: true},
```

Comment:
Following the Airbnb style guide, I believe two things are true (let me know if I am mistaken here):

1. An object should have spaces between the start of the object literal (curly brace) and the first key, as well as between the last value and the closing brace (i.e., { key: value })

2. Object literals should span multiple lines, with an indentation to start the first key below